spacy/tests/tokenizer/test_explain.py

```diff
@@ -123,6 +135,9 @@ def test_tokenizer_explain_fuzzy(lang: str, sentence: str) -> None:
     """
 
     tokenizer: Tokenizer = spacy.blank(lang).tokenizer
-    tokens = [t.text for t in tokenizer(sentence) if not t.is_space]
+    # Tokenizer.explain is not intended to handle whitespace or control
+    # characters in the same way as Tokenizer
```

Comment:
Out of curiosity, why is that?

Reply:
Because it wasn't a priority when implementing it. The main thing was to make it easier to debug all the tokenizer regexes, which mostly operate on a single whitespace-bounded string. (Not 100% given the phrase matcher for special cases, which is what is affected here.)